### DISCUSSION TITLE
Column width flexibility

### DISCUSSION THREAD

**Discussion by blink-twice** [githubocto/flat-action#42] [Q&A] [answered]
Is there a way to fit column widths to the name of the column in the dataset or resize them? It currently uses width based on the values in the column.  

![image](https://user-images.githubusercontent.com/64839036/120619616-c29ba800-c479-11eb-8c9e-8739c8b666f8.png)

[Link to view](https://flatgithub.com/IDFC-Institute/mumbai-covid-data?filename=data%2Fmumbai_c19_tests_by_type.csv&filters=&sha=4643b31727f3035e4b36fd5c1f1f2671bbb0fca5&sort=Date%2Cdesc&stickyColumnName=Total_tests)

 This makes it hard for users to comprehend what's on the screen and requires that a user hover over the column name to understand what its about, even if there are very few columns in the dataset.

**Comment by Wattenberger** [comment, marked as answer]
Thanks for flagging - agreed that it makes sense to use the column names in the width calculation. Updated!
